pages/common/hugo.md

```diff
@@ -0,0 +1,31 @@
+# hugo
+
+> Hugo is a Fast and Flexible Static Site Generator.
+
+- Create a new hugo site:
+
+`hugo new site {{path/to/site}}`
+
+- Create a new hugo theme (you may also download one from [https://themes.gohugo.io/]()):
+
+`hugo new theme {{theme_name}}`
+
+- Create a new page:
+
+`hugo new {{section_name}}/{{filename}}{{.ext}}`
+
+- Build your site to the `./public/` directory:
+
+`hugo`
+
+- Build your site to a different destination:
+
+`hugo --destination {{path/to/destination}}`
+
+- Build your site, start up a webserver to serve it, and watch for changes:
+
+`hugo server`
+
+- Build and serve content marked as draft:
```

Comment:
When you first create a "page" it is automatically marked as a "draft". This stands for a page that is a work in progress since you are still drafting it up, so to speak. I included this example because "drafts" are _not_ built when you run `hugo` or `hugo server` and it is kind of confusing to have written a page but have forgotten to toggle the draft status off (it's a variable). By including `--buildDrafts` "draft" pages will also be built.